### DISCUSSION TITLE
Recommended way to offer bank transfer / prepay without gateway integration

### DISCUSSION THREAD

**Discussion by genox** [saleor/saleor#13324] [Q&A]
Hi,

I was wondering..  it is common for some stores to offer a simple "transfer to our IBAN and we will ship as soon as we receive your money" payment method. How would I go about integrating this on saleor?

From what I can see now I could extend the offered payment gateways in the client with a new one and complete the checkout without initialising a payment gateway process, but it looks like checkoutComplete requires payment gateway data in any case.

Is this even considered a viable way to handle checkouts? Or should I instead entirely rely on something like GiroPay from stripe?

**Comment by korycins** [comment]
Hey @genox, we are already working on the possibility to create order from checkoutComplete that is not paid. It sounds like a perfect match for the described case. 
We will allow you to create the unpaid order, and you or your integration can mark an order as paid later. 
Here is a RFC for this part: https://github.com/saleor/saleor/issues/13073

**Reply by genox** [reply]
yeah, that would be perfect. thank you for the heads-up.
